effortless-chef-io/effortless-config.md

```diff
@@ -4,43 +4,67 @@ Effortless Config is the pattern for managing your Chef Infra workloads. It uses
 
 ## Patterns
 
-### Chef Repo Cookbook pattern
+### Chef Repo Cookbook Pattern
 
-This pattern is used to build policyfiles using the chef_repo pattern for organizing your cookbooks. The chef-repo pattern can be found [here](https://docs.chef.io/chef_repo/).
+This pattern uses the [chef-repo](https://docs.chef.io/chef_repo/) to store and organize everything you need to define your infrastructure with Chef Infra, including:
 
-> Note: You can view and example of this pattern [here](https://github.com/chef/effortless/tree/master/examples/effortless_config/chef_repo_pattern).
+- Cookbooks (including recipes, attributes, custom resources, libraries, and templates)
+- Data bags
+- Policyfiles
+  
+The Chef Effortless GitHub repository has an [example chef-repo](https://github.com/chef/effortless/tree/master/examples/effortless_config/chef_repo_pattern) for you to explore.
 
-1. To use this pattern navigate to the chef-repo you want to use
-   ```
+1. To use this pattern, navigate to the chef-repo directory that you want to use:
+
+   ```bash
    cd chef-repo
    ```
-1. Make a habitat directory
-   ```
+
+1. Create a `habitat` directory from the command line with:
+
+   ```bash
    mkdir habitat
    ```
+
 1. Make a plan file
-   
-   > Notes: For a cookbook targeting windows use a `plan.ps1` for Linux use a `plan.sh` if your cookbook targets both windows and linux you can have both a `plan.ps1` and a `plan.sh` in your habitat directory.
-   ```
+
+   Make  a `plan.ps1` for a cookbook that targets Microsoft Windows systems and a `plan.sh` for a cookbook that targets Linux systems. You can have both a `plan.ps1` and a `plan.sh` in the `habitat` directory for cookbooks that target both systems. Create a Linux plan:
+
+   ```bash
    touch plan.sh
    ```
-1. Add some information about your cookbok to your plan
-   plan.sh
+
+1. Add information about the cookbook to the plan
+
+   Add this information to the `plan.sh` file:
+
    ```bash
-   pkg_name=<NAME FOR YOUR POLICYFILE>
-   pkg_origin=<YOUR ORIGIN>
+   pkg_name=<NAME FOR MY POLICYFILE>
+   pkg_origin=<MY ORIGIN>
    pkg_version="0.1.0"
-   pkg_maintainer="YOUR NAME AND EMAIL"
+   pkg_maintainer="MY NAME AND EMAIL"
    pkg_license=("Apache-2.0")
    pkg_scaffolding="chef/scaffolding-chef-infra"
    pkg_svc_user=("root")
-   scaffold_policy_name="<YOUR POLICYFILE NAME>"
+   scaffold_policy_name="<MY POLICYFILE NAME>"
```

Comment:
@kagarmoe What about this one? :)

Reply:
There's another on line 43

Reply:
Good catch! `my_policyfile` because we can't have spaces in the policyfile name

Reply:
And it goes with the example you put further down. BRB gonna update the other PR, too.